pkgs/applications/networking/instant-messengers/wire-desktop/default.nix

```
@@ -67,39 +60,34 @@ let
     };
 
     dontBuild = true;
-    dontPatchELF = true;
     dontConfigure = true;
+    dontPatchELF = true;
+    dontWrapGApps = true;
 
-    nativeBuildInputs = [ dpkg ];
-    rpath = stdenv.lib.makeLibraryPath [
-      alsaLib at-spi2-atk atk cairo cups dbus expat fontconfig freetype
-      gdk-pixbuf glib gtk3 hunspell libX11 libXScrnSaver libXcomposite
-      libXcursor libXdamage libXext libXfixes libXi libXrandr libXrender
-      libXtst libnotify libuuid nspr nss pango pciutils pulseaudio
-      stdenv.cc.cc udev xdg_utils xorg.libxcb
-    ];
+    buildInputs = atomEnv.packages ;
+
+    nativeBuildInputs = [ dpkg makeWrapper autoPatchelfHook wrapGAppsHook ];
```

```suggestion
    nativeBuildInputs = [ dpkg autoPatchelfHook wrapGAppsHook ];
```

and if you could order this attr before `buildInputs`, and use one list item per line.